apps/reference/docs/guides/auth/auth-helpers/auth-ui.mdx

```diff
@@ -1,28 +1,38 @@
 ---
-id: auth-ui-overview
+id: auth-ui
 title: Auth UI
-description: Securing your user data with Row Level Security.
+description: A prebuilt, customizable React component for authenticating users.
 ---
 
-# Auth UI
+Auth UI is a pre-built React component for authenticating users.
+It supports custom themes and extensible styles to match your brand and aesthetic.
 
-Auth UI is a React component that provides a simple way to authenticate users.
+<video width="99%" muted playsInline controls="true">
+  <source
+    src="https://supabase.com/images/blog/lw5-one-more/auth-ui-demo.mp4"
```

Comment:
Couldn't figure out how to properly reference https://github.com/supabase/supabase/blob/master/apps/www/public/images/blog/lw5-one-more/auth-ui-demo.mp4 from `apps/reference/docs/guides`...

However, it'd be even cooler to embed the widget on supabase.com/auth instead of this video 😄